ui/src/components/VolumeDetailCard.js

```diff
@@ -281,9 +299,23 @@ const VolumeDetailCard = (props) => {
             data-cy="delete_volume_button"
           ></DeleteButton>
         </DeleteButtonContainer>
+        {
+          alertlist &&
+          <AlertsCounterContainer>
+            <VolumeSectionTitle>
+              {intl.translate('active_alerts')}
+              &nbsp;
+            <TextBadge>{alertlist?.length}</TextBadge>
+            </VolumeSectionTitle>
+            <ActiveAlertsCounter
+              alerts={alertlist}
+              baseLink={`${match.url}/${name}/alerts`}
+            />
+          </AlertsCounterContainer>
+        }
         {condition === VOLUME_CONDITION_LINK && (
           <VolumeUsage>
-            <VolumeUsageTitle>Volume Usage</VolumeUsageTitle>
+            <VolumeSectionTitle>{intl.translate('volume_usage')}</VolumeSectionTitle>
```

Comment:
The title should be just `Usage` according to the design.
And thanks for the translations.